domains/zwu.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "wu000168",
+        "email": "wuzed@seas.upenn.edu"
+    },
+    "record": {
+        "CNAME": "https://github.com/wu000168"
```

`CNAME` should not contain protocol
```suggestion
        "URL": "https://github.com/wu000168"
```